robustirc-bridge/bridge.go

```diff
@@ -41,6 +48,41 @@ var (
 // - for resuming sessions (later): the last seen message id, perhaps setup messages (JOINs, MODEs, …)
 // for hosted mode, this state is stored per-nickname, ideally encrypted with password
 
+func prefixMotd(msg string) string {
+	// The user chose to not inject a MOTD.
+	if *motdPath == "" {
+		return msg
+	}
+
+	sep := strings.Index(msg[1:], ":")
+	if sep == -1 {
+		return msg
+	}
+
+	prefix := msg[:sep+2] + "- "
+
+	f, err := os.Open(*motdPath)
```

Comment:
I’m not trying to save the RAM :). I thought it’d be nice to update the MOTD without needing to introduce some command for the bridge to re-read it. You obviously can’t restart the bridge, as it’s supposed to be always-running.